go.mod

```diff
@@ -1,60 +1,64 @@
 module github.com/rancher/rancher
 
-go 1.19
+go 1.20
 
 // on release remove this wrangler replace and use the latest tag
-replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230705223603-201b4da5bdaf
```

Comment:
Wrangler now has a tag for this commit that can be used instead of this replace https://github.com/rancher/wrangler/tree/v2.0.1

Reply:
actually there is an issue with adding that version in go mod. while trying to do go get we i am facing the below error
```
go: github.com/rancher/wrangler@v2.0.1: invalid version: module contains a go.mod file, so module path must match major version ("github.com/rancher/wrangler/v2")
```
so i think go requires module path to be updated when we move from v1 to v2 and that's why update in wrangler will be needed. The deadline for 1.27 was near so we decided to go with the commit tag for now.
cc: @kinarashah

Reply:
@KevinJoiner Yep, my understanding was we branched wrangler to manage branches but this isn't technically a new module for us to treat it as new v2 and change usage to `rancher/wrangler/v2`. If that's not true, we could update it in another PR to follow up.

Reply:
Ah, I forgot that go requires you to change the path. We went to v2.0.1 because this version of wrangler has multiple breaking changes, so I think we need to treat it as a new module, but as you suggested we can update the paths and handle that in a separate PR.

Reply:
I have a PR up for this now https://github.com/rancher/wrangler/pull/317.